src/formatters/deployProgressBarFormatter.ts

```diff
@@ -68,7 +73,6 @@ export class DeployProgressBarFormatter extends ProgressFormatter {
 
   // used to initialize the progress bar
   protected initProgressBar(): void {
```

Comment:
now that `progressBar` is initialized when defined this can method can be removed.

Reply:
removed